templates.md

```diff
@@ -138,6 +138,16 @@ To overwrite a section entirely, you may use the `overwrite` statement:
 
 	{{-- This comment will not be in the rendered HTML --}}
 
+#### Service injection
+We can use `@inject` to resolve services from the service container.
+
+	@inject('html', 'HtmlBuilder')
+
+	<div>
+		{{ $html->image('/assets/images/laravel.png') }}
```

Comment:
That is the wrong blade syntax. That would result in escaped html dumped on the page.